From 3469731469f96c4de0fa4880f5ecc66f4f6f9985 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Fri, 27 Oct 2017 19:08:20 +0000
Subject: [PATCH] OS-6424 dladm show-phys -P speaks gibberish Reviewed by: Dan
 McDonald <danmcd@joyent.com> Approved by: Dan McDonald <danmcd@joyent.com>

---
 usr/src/cmd/dladm/dladm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/usr/src/cmd/dladm/dladm.c b/usr/src/cmd/dladm/dladm.c
index a339dab76e..c4fe45cf40 100644
--- a/usr/src/cmd/dladm/dladm.c
+++ b/usr/src/cmd/dladm/dladm.c
@@ -3829,6 +3829,7 @@ print_phys_default(show_state_t *state, datalink_id_t linkid,
 	if (status != DLADM_STATUS_OK)
 		goto done;
 
+	bzero(&pattr, sizeof (pattr));
 	(void) snprintf(pattr.link_phys_device,
 	    sizeof (pattr.link_phys_device), "%s", dpa.dp_dev);
 	(void) dladm_media2str(media, pattr.link_phys_media);
-- 
2.21.0

